lgt8f/boards.txt

```diff
@@ -35,26 +35,33 @@ menu.upload_speed=Upload speed
 328.menu.arduino_isp.enable.build.SERIAL_RX_BUFFER_SIZE=250
 
 # Clock source
-328.menu.clock_source.internal=Internal
+328.menu.clock_source.internal=Internal 32MHz
 328.menu.clock_source.internal.build.clock_source=1
 328.menu.clock_source.internal.build.f_osc=32000000L
-328.menu.clock_source.external=External (assumes 32MHz crystal)
-328.menu.clock_source.external.build.clock_source=2
-328.menu.clock_source.external.build.f_osc=32000000L
+328.menu.clock_source.external_32=External 32MHz
+328.menu.clock_source.external_32.build.clock_source=2
+328.menu.clock_source.external_32.build.f_osc=32000000L
+328.menu.clock_source.external_16=External 16MHz
+328.menu.clock_source.external_16.build.clock_source=2
+328.menu.clock_source.external_16.build.f_osc=16000000L
+328.menu.clock_source.external_12=External 12MHz
+328.menu.clock_source.external_12.build.clock_source=2
+328.menu.clock_source.external_12.build.f_osc=12000000L
+
+# Clock divider
+328.menu.clock_div.1=1x
```

Comment:
In my opinion the divider menu will look better without 'x'
```
1
2
4
8
16
32
```

Reply:
updated